database/seeds/DatabaseSeeder.php

```diff
@@ -12,6 +12,7 @@ public function run()
             'twitter' => 'john',
             'country_code' => 'USA',
             'password' => \Illuminate\Support\Facades\Hash::make('secret'),
+            'mastodon' => 'john'
```

Comment:
This would not be a valid username since it doesn't have the server name in it.

Here's the actual regex:
https://github.com/mastodon/mastodon/blob/1e3b19230a48174acf524cf1a9f5a498e220ea7d/app/models/account.rb#L66

That same regex may be overkill; anything that matches [filer_var](https://www.php.net/manual/en/filter.examples.validation.php) (source [here](https://github.com/php/php-src/blob/655f116be57b46efe32221d7adfec6d6b81eeece/ext/filter/logical_filters.c#L651-L718)) would probably work.

Reply:
Thanks! I personally don't have Mastodon so wasn't 100% how usernames worked. I think my latest push to this branch makes the seeder data more accurate though!